applications/nrf_desktop/src/modules/Kconfig.watchdog

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Watchdog settings"
+
+config DESKTOP_WATCHDOG_ENABLE
+    bool "Enable watchdog support"
+    depends on WATCHDOG
+    help
+      Enable watchdog to reset the device in case of problems
+
+if DESKTOP_WATCHDOG_ENABLE
```

Comment:
Because log level sourced config also depends on DESKTOP_WATCHDOG_ENABLE the I decided to enclose "Watchdog timeout" entry and sourced template with the if-endif instead of add "depends on" to "Watchdog timeout" and separately enclose sourced template.

Reply:
ok